src/awssdk_driver.rs

```diff
@@ -38,6 +39,8 @@ impl<C> QldbSessionSdk<C> {
         }
     }
 }
+const DRIVER_VERSION: &'static str = "QLDB Driver for Rust v0.1.0";
+const SHELL_VERSION: &'static str = "QLDB Shell for Rust v2.0.0";
```

Comment:
Please can you use the same mechanism as was previously in use, where we don't need to manually keep this up to date?

Reply:
https://github.com/awslabs/amazon-qldb-shell/blob/b18304949f98f26b29428400275ddb86f0c68f5e/src/rusoto_driver.rs#L96-L101